src/components/ha-tooltip.ts

```diff
@@ -22,10 +22,10 @@ export class HaTooltip extends SlTooltip {
       :host {
         --sl-tooltip-background-color: var(--secondary-background-color);
         --sl-tooltip-color: var(--primary-text-color);
-        --sl-tooltip-font-family: Roboto, sans-serif;
-        --sl-tooltip-font-size: 12px;
-        --sl-tooltip-font-weight: normal;
-        --sl-tooltip-line-height: 1;
+        --sl-tooltip-font-family: var(--ha-tooltip-font-family);
```

Comment:
I'd rather leave these undefined personally if not set in a theme and use the default directly:
```suggestion
        --sl-tooltip-font-family: var(--ha-tooltip-font-family, var(--ha-font-family-body));
```